### PR TITLE
chore: update codeowners file to reflect the updated repository structure

### PR DESCRIPTION
**Pull request by janechu**
# Description

The `CODEOWNERS` file had fallen out of date, this change adds missing packages, removes packages that are no longer present and updates the pathing for packages.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -74,14 +22,42 @@
...
+# Tooling
+/packages/tooling/fast-figma-plugin-msft/ @nicholasrice @bheston @e-vanaubrey @Jeremy-Knudsen
+/packages/tooling/fast-tooling-react/ @janechu @nicholasrice @chrisdholt
+/packages/tooling/fast-tooling/ @janechu @nicholasrice @chrisdholt
+
+# Utilities
+/packages/utilities/fast-animation/ @nicholasrice @chrisdholt
+/packages/utilities/fast-colors/ @nicholasrice @chrisdholt
+/packages/utilities/fast-eslint-rules/ @nicholasrice @janechu @chrisdholt
+/packages/utilities/fast-web-utilities/ @janechu @chrisdholt @nicholasrice
```
Comment (janechu) [inline .github/CODEOWNERS:59]: I've left off the `fast-tslint-rules` package as that is in process of being deprecated anyway.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -74,14 +22,42 @@
...
+/packages/react/fast-jss-manager/ @nicholasrice @janechu @chrisdholt
+/packages/react/fast-jss-manager-react/ @nicholasrice @janechu @chrisdholt
+/packages/react/fast-jss-utilities/ @nicholasrice @janechu @chrisdholt
+/packages/react/fast-layouts-react/ @chrisdholt @nicholasrice
+/packages/react/fast-react-utilities/ @chrisdholt @nicholasrice
+/packages/react/fast-storybook-design-system-addon/ @nicholasrice @chrisdholt @janechu
+/packages/react/fast-storybook-presets/ @nicholasrice @chrisdholt @janechu
+
+# Tooling
+/packages/tooling/fast-figma-plugin-msft/ @nicholasrice @bheston @e-vanaubrey @Jeremy-Knudsen
```
Comment (nicholasrice) [inline .github/CODEOWNERS:51]:
```suggestion
/packages/tooling/fast-figma-plugin-msft/ @nicholasrice @bheston 
```

Those didn't work anyway so we can remove

Comment (chrisdholt) [reply .github/CODEOWNERS:51]: we should likely add one more here just to ensure sign off can take place if you or @bheston are unavailable.

Comment (janechu) [reply .github/CODEOWNERS]: Done

**Pull request merged**